etna/articles/models.py

```diff
@@ -260,6 +261,14 @@ class RecordArticlePage(TopicalPageMixin, ContentWarningMixin, BasePageWithIntro
         help_text="Link to an external print on demand service",
     )
 
+    promoted_links = StreamField(
+        [("promoted_link", AuthorPromotedLinkBlock())],
+        max_num=3,
+        blank=True,
+        null=True,
+        use_json_field=True,
+    )
```

Comment:
If this is optional, would this make sense as an option in the existing `body` `StreamField`?

Reply:
Hi @ababic, there's no other StreamFields on this page - there's a "body" `MultiFieldPanel` but this is used for the stuff relevant to the `RecordArticle` such as the record info, and links to the image/print services :)